bin/MorphingMSSMvsSM.cpp

```diff
@@ -2373,7 +2422,6 @@ int main(int argc, char **argv) {
       ws.import(*i_frac, RooFit::RecycleConflictNodes());
     }
     fractions_sm.Close();
-  }
```

Comment:
Is this removal of the brace correct? I would fear the code will not compile with this modification.

Reply:
It seems it is and the previous version of the code did not compile. So everything is fine with this PR.